src/web_app/js/appcontroller_test.js

```diff
@@ -80,10 +85,16 @@ describe('AppControllerTest', function() {
         .toBeFalsy();
   });
 
-  it('Hide UI after clicking the join button', function() {
+  it('Hide UI after clicking the join button', function(done) {
     // Verifies that the UI is hidden after clicking the button.
-    $(UI_CONSTANTS.confirmJoinButton).click();
-    expect($(UI_CONSTANTS.confirmJoinDiv).classList.contains('hidden'))
-        .toBeTruthy();
+    // Need to wait a few seconds for the element to update it's class list.
```

Comment:
Yeah I believe the classList for the element is not updated fast enough. Will add more info.